### PR TITLE
docs: How-to guides review

### LINKED ISSUE #5332: [DOCS]  Review How-to guide code snippets

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

Closes #5332 

**Type of change**

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (change restructuring the codebase without changing functionality)
- Improvement (change adding some improvement to an existing functionality)
- Documentation update

**How Has This Been Tested**

**Checklist**

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: argilla/docs/how_to_guides/record.md [hunk 1]
```diff
@@ -468,23 +468,23 @@ dataset.records.log(records=updated_data)
...
 
     ```python
     updated_records = []
 
     for record in dataset.records(with_vectors=True):
 
-        record.vectors["new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
-        record.vectors["v"] = [ 0.1, 0.2, 0.3 ]
+        record.vectors["my_vector"] = [ 0, 1, 2, 3, 4, 5 ]
+        record.metadata["my_new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
```
Comment (frascuchon) [inline argilla/docs/how_to_guides/record.md]:
```suggestion
        record.vectors["my_new_vector"] = [ 0, 1, 2, 3, 4, 5 ]
```

**Comment by burtenshaw** [comment]
@sdiazlor @davidberenstein1957 Is there a way to configure example code snippets in docstrings to be consistent with those in markdown?

I see `>>>` the only solution I could find was to add ```python and `>>>`.

Screenshot 2024-07-30 at 13 08 47

**Review by frascuchon — APPROVED** [review]

**Comment by sdiazlor** [comment]
> @sdiazlor @davidberenstein1957 Is there a way to configure example code snippets in docstrings to be consistent with those in markdown?
> 
> I see `>>>` the only solution I could find was to add ```python and `>>>`.
> 
I'd need to make some research, maybe there is a way using the mkdocstrings dependency, but I'm not sure.

**Comment by burtenshaw** [comment]
> > @sdiazlor @davidberenstein1957 Is there a way to configure example code snippets in docstrings to be consistent with those in markdown?
> > I see `>>>` the only solution I could find was to add ```python and `>>>`.
> 
> I'd need to make some research, maybe there is a way using the mkdocstrings dependency, but I'm not sure.

No worries. It's not major, just in case there was a simple solution.

**Pull request merged**